tests/acceptance/features/apiSpacesDavOperation/moveByFileId.feature

```diff
@@ -150,6 +150,31 @@ Feature: moving/renaming file using file id
       | /dav/spaces/<<FILEID>>            |
 
 
+  Scenario Outline: Move a file from sub-folder to root folder inside Shares space
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "/folder"
+    And user "Alice" has created folder "folder/sub-folder"
+    And user "Alice" has uploaded file with content "some data" to "/folder/sub-folder/test.txt"
+    And we save it into "FILEID"
+    And user "Alice" has shared folder "/folder" with user "Brian" with permissions "all"
+
+    When user "Brian" moves a file "Shares/folder/sub-folder/test.txt" into "Shares/folder" inside space "Shares" using file-id path "<dav-path>"
+    Then the HTTP status code should be "502"
```

Comment:
```suggestion
    Then the HTTP status code should be "403"
```

Reply:
`502` does not seem appropriate. but there was some discussion regarding this status code. @nabim777 can you please mention that issue here.

Reply:
https://github.com/owncloud/ocis/pull/7535#discussion_r1366904945